readme_mod_13_logging.md

```diff
@@ -0,0 +1,87 @@
+# Setting Up Logging with Loggly
+
+## Introduction
+
+This guide will walk you through the steps to set up logging in your Python application using Loggly, a cloud-based logging service.
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+- Python installed on your system
+- Pip package manager
+- An active Loggly account
+- Loggly token (API Key)
+
+## Step 1: Install Required Packages
+
+First, install the necessary Python package for integrating with Loggly. Open your terminal or command prompt and run:
+
+```bash
+pip install loggly-python-handler
+```
```

Comment:
Remember we should be sticking with poetry not pip!
`poetry add loggly-python-handler`